pkgs/development/python-modules/sphinxcontrib-seqdiag/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, mock
+, sphinx-testing
+, sphinx
+, blockdiag
+, seqdiag
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-seqdiag";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-THJ1ra/W2X/lQaDjGbL27VMn0lWPJApwgKMrPhL0JY0=";
+  };
+
+  buildInputs = [ mock sphinx-testing ];
```

Comment:
Are those required for tests? If they are not required at all please remove them from the setup.py.

Reply:
There are tests for these packages, but I wasn't able to get `unittest` to run them after some experimentation, that appears to have been the conclusion in the existing `sphinxcontrib-blockdiag` package too:

https://github.com/NixOS/nixpkgs/blob/42add18071279a9bfbffcf807976ce72925826e1/pkgs/development/python-modules/sphinxcontrib-blockdiag/default.nix#L23-L24

I've removed these inputs and added a `pythonImportsCheck` for each package.

Reply:
Those are wrong. Test depedencies go to checkInputs.

Reply:
They aren't there anymore, I've removed them.